### DISCUSSION TITLE
Question: What goes in issues and what goes in Discussions?

### DISCUSSION THREAD

**Discussion by isakube** [KadeArchive/Kade-Engine#424] [Q&A] [answered]
i'm just trying to make a .md that states what is and is not an Issue for the Kade Engine repo and I can't really figure out the line inbetween Issues and Discussions.

**Comment by isakube** [comment, marked as answer]
lol i read some Issue's comments

**Comment by puyoxyz** [comment]
noone is gonna read it anyways lol don't bother; and we can convert issues to discussions

**Reply by isakube** [reply]
yeah
